src/psyclone/psyir/nodes/acc_directives.py

```diff
@@ -273,14 +295,43 @@ def data_on_device(self, parent):
         '''
 
 
-class ACCParallelDirective(ACCRegionDirective):
+class ACCParallelDirective(ACCRegionDirective, ACCAsyncMixin):
     '''
     Class representing the !$ACC PARALLEL directive of OpenACC
     in the PSyIR. By default it includes the 'DEFAULT(PRESENT)' clause which
     means this node must either come after an EnterDataDirective or within
     a DataDirective.
 
+    :param children: the PSyIR nodes to be enclosed in the ACC parallel region\
+                         and which are therefore children of this node.
+        :type children: List[:py:class:`psyclone.psyir.nodes.Node`]
```

Comment:
Please could you un-indent these lines (i.e. line them up with L305) as the build of the Reference guide is now giving a lot of new errors. (cd PSyclone/doc/reference_guide; make html - there should be just 17 warnings.)